MAINTAINERS.yml

```diff
@@ -0,0 +1,384 @@
+    labels:
+        - "area: Build System"
+
+"Display drivers":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/display/
+        - dts/bindings/display/
+        - include/display.h
+        - include/display/
+        - include/drivers/display.h
+    labels:
+        - "area: Display"
+
+Documentation:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - dbkinder
+    files:
+        - doc/
+    files-regex:
+        - \.rst$
+        - /doc/
+    labels:
+        - "area: Documentation"
+
+Flash:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - drivers/flash/
+        - dts/bindings/flash_controller/
+        - include/drivers/flash.h
+        - include/flash.h
+        - samples/drivers/flash_shell/
+    labels:
+        - "area: Flash"
+
+Kconfig:
+    status: maintained
+    maintainers:
+        - ulfalizer
+    files:
+        - scripts/kconfig/
+    files-regex:
+        - Kconfig[\w._-]*$
+    labels:
+        - "area: Kconfig"
+    description: >-
+        See https://docs.zephyrproject.org/latest/guides/kconfig/index.html and
+        https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#default-board-configuration
+
+Logging:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nordic-krch
+    files:
+        - include/logging/
+        - samples/subsys/logging/
+        - subsys/logging/
+        - tests/subsys/logging/
+    labels:
+        - "area: Logging"
+
+"Native POSIX and POSIX arch":
+    status: maintained
+    maintainers:
+        - aescolar
+    files:
+        - arch/posix/
+        - boards/posix/native_posix/
+        - drivers/*/*native_posix*
+        - drivers/*/*/*native_posix*
+        - dts/posix/
+        - include/arch/posix/
+        - scripts/valgrind.supp
+        - soc/posix/
+        - tests/boards/native_posix/
+    labels:
+        - "area: native port"
+    description: >-
+        POSIX architecture and SOC, native_posix board, and related drivers
+
+"Native POSIX flash driver":
+    status: maintained
+    collaborators:
+         - vanwinkeljan
+    files:
+         - drivers/flash/flash_native_posix.c
+
+"nRF52 BSIM":
+    status: maintained
+    maintainers:
+        - aescolar
+    files:
+        - boards/posix/nrf52_bsim/
+    labels:
+        - "platform: nrf52_bsim"
+
+"RISCV arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nategraff-sifive
+    collaborators:
+        - mgielda
+    files:
+        - arch/riscv/
+    labels:
+        - "area: RISCV"
+
+Sanitycheck:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nashif
+    files:
+        - scripts/sanitycheck
+        - scripts/sanity_chk/
+    labels:
+        - "area: Sanitycheck"
+
+"Settings subsystem":
```

Comment:
here using "subsystem", below and in other places, we do not, so we should be consistent, just call it "Settings", no need for subsystem.

Reply:
No mention of subsystems anywhere now.